### PR TITLE
Cutadapt 2 and initial recipe for dnaio

### PR DESCRIPTION
**Pull request by marcelm**
* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [X] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

**Comment by marcelm** [comment]
Ah, the joy of waiting for the macOS build to finish ...

File: recipes/dnaio/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  name: dnaio
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/1f/be/73d03ccdc8f721b938c5201d3031c676fa164fc6641f6d722160e5b63036/dnaio-0.3.tar.gz
+  sha256: 47e4449affad0981978fe986684fc0d9c39736f05a157f6cf80e54dae0a92638
+
+build:
+  number: 0
+  script: pip install --no-deps .
```
Comment (bgruening) [inline recipes/dnaio/meta.yaml]: can you use it like that: http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#use-pip

File: recipes/cutadapt/meta.yaml [hunk 2]
```diff
@@ -1,28 +1,32 @@
...
 source:
-  url: https://files.pythonhosted.org/packages/fd/4d/3ac2947d36e2d56ce1513dc4a53d3a45e520d7ccfb43af9ba85408e44c69/cutadapt-1.18.tar.gz
-  sha256: 17aabf9b19d09a426d96030a83ad003c97b26dba9d45bf5570d33088fcd533f9
+  url: https://files.pythonhosted.org/packages/f2/bd/c4054b10cff0a595f6343edf874b98d6bf568703f0bacb0bfd2967ef662a/cutadapt-2.0.tar.gz
+  sha256: e2f94acdcba745de8c9c44106e99b22975439c9a7ec236aa85f7c3baea839fd1
 
 build:
-  number: 1
+  number: 0
   script: pip install --no-deps .
```
Comment (bgruening) [inline recipes/cutadapt/meta.yaml]: Same here, can you adjust it to http://conda-forge.org/docs/maintainer/adding_pkgs.html?highlight=pip%20install#use-pip

File: recipes/dnaio/meta.yaml (hunk 1 above)
Comment (marcelm) [reply recipes/dnaio/meta.yaml]: Absolutely

**Comment by marcelm** [comment]
Thanks :-)

**Pull request merged**